site/content/docs/10-exporting.md

```diff
@@ -44,7 +44,17 @@ You can also add a script to your package.json...
 
 When you run `sapper export`, Sapper first builds a production version of your app, as though you had run `sapper build`, and copies the contents of your `static` folder to the destination. It then starts the server, and navigates to the root of your app. From there, it follows any `<a>` elements it finds, and captures any data served by the app.
 
-Because of this, any pages you want to be included in the exported site must either be reachable by `<a>` elements or added to the `--entry` option of the `sapper export` command. Additionally, any non-page routes should be requested in `preload`, *not* in `onMount` or elsewhere.
+Because of this, any pages you want to be included in the exported site must either be reachable by `<a>` elements or added to the `--entry` option of the `sapper export` command.
+
+The `--entry` option expects a string of space-separated values. Examples:
+
+```bash
+sapper export --entry "some-page some-other-page"
+
+sapper export --entry single-entry
+```
+
+Additionally, any non-page routes should be requested in `preload`, *not* in `onMount` or elsewhere.
```

Comment:
I think it would be useful to explain why

Reply:
I actually no longer remember why myself, and don’t really have time to dig into it again. But I agree it would be useful.

Reply:
I think onMount is run only in the browser so it would not be picked up if you wanted onMount resources exported.

Reply:
Could you also clarify the example with exposition for what files are there? That is, what does `some-page` map to? `src/routes/some-page.svelte` or `src/routes/some-page/index.svelte` or potentially both?